packages/fiber/tests/core/renderer.test.tsx

```diff
@@ -603,17 +613,10 @@ describe('renderer', () => {
   })
 
   it('will render components that are extended', async () => {
-    class MyColor extends THREE.Color {
```

Comment:
Not sure about this. Can you explain why you think modifying the test the way you have is better?

Reply:
As far as I understand, we'd like to test whether the `extend` function works. We can still do this even if the `MyColor` class is declared externally. This is how I was able to add the `myColor` property to the `IntrinsicElements` interface,